src/components/NFTCover.vue

```diff
@@ -101,6 +118,15 @@ export default {
     isShowImage() {
       return this.src && !this.isError;
     },
+    bookSpineStyle() {
+      if (this.spineColor1 && this.spineColor2)
+        return [
+          'bg-gradient-to-b',
+          `from-[${this.spineColor1}]`,
+          `to-[${this.spineColor2}]`,
```

Comment:
i dont think these dynamic class names will be recognized for Tailwind, need to set CSS in style explicitly